Rakefile

```diff
@@ -1,2 +1,9 @@
-require "bundler"
+require 'bundler'
+require 'rubocop/rake_task'
+require 'rspec/core/rake_task'
+
 Bundler::GemHelper.install_tasks
+RSpec::Core::RakeTask.new(:spec)
+RuboCop::RakeTask.new
+
+task default: %i(rubocop spec)
```

Comment:
Please let’s not run rubocop within travis. Let’s  use HoundCI or similar instead. We don’t want to have Travis failing because of rubocop. Then we can make travis a mandatory check, while Hound is optional.